perf/sawtooth_perf/src/batch_submit.rs

```diff
@@ -222,3 +224,37 @@ impl<'a> BatchListFeeder<'a> {
         Ok(Some(batch_list))
     }
 }
+
+pub trait BatchListIter {
```

Comment:
This should just be the `std::iter::Iterator`

Reply:
Sounds good. I will update.